client/tsconfig.json

```diff
@@ -16,9 +16,10 @@
     "incremental": true,
     "baseUrl": ".",
     "paths": {
-      "@/*": ["./*"]
+      "@/*": ["./*"],
+      "~/*": ["../server/src/*"]
```

Comment:
this is kinda hacky, didn't take the time to set up a base tsconfig.json properly

Reply:
Can you please add this as a comment in the code here. A pointer to an existing article (or new article/doc in our wiki) can be included to keep the comment brief.